### PR TITLE
Add new intro videos

### PR DESCRIPTION
**Pull request by glassnotes**
**Title:** Add new intro videos

**Summary:** Videos from [this playlist](https://www.youtube.com/playlist?list=PL_hJxz_HrXxsQNJHWp10up8x-hwd5uwr0) have been added to the videos page. (@co9olguy / @josh146 I've put this here so you can merge at your leisure when you'd like to announce the videos).

**Relevant references:**  None

**Possible Drawbacks:** None

**Related GitHub Issues:** #171

### CODE REVIEW

**Review by josh146 — APPROVED** [review]
Nathan dominating 📹 

Quick thought: should we have a separate section just for this video series, rather than lumping them with the other videos? Since these 4 are much more interconnected and should be watched as a series

**Comment by glassnotes** [comment]
> Nathan dominating video_camera
> 
> Quick thought: should we have a separate section just for this video series, rather than lumping them with the other videos? Since these 4 are much more interconnected and should be watched as a series

Yeah I was thinking about that. Another option is to have a subtitle with numbering, like "QML Intro Series 1/4". But also it might be nice to partition things into sections by topic anyways, since there are quite a few videos now on the page.

**Comment by co9olguy** [comment]
I agree that the videos page needs a refresh. Perhaps timed to align with QHACK?

File: videos.rst [hunk 1]
```diff
@@ -98,6 +98,42 @@ Sit back and explore quantum machine learning and quantum programming with our c
...
+    Explore the relationships between deep learning, differentiable
+    programming, and quantum computing. Learn how quantum circuits can be
+    trained just like neural networks!
+
+.. youtube:: McgBeSVIGus
+    :title: Automatic Differentiation of Quantum Circuits
+    :author: Nathan Killoran (Xanadu)
+
+    Dive into the world of quantum gradients. Learn how to apply the parameter shift
+    rules, and how partial derivatives of quantum circuits can be computed
```
Comment (co9olguy) [inline videos.rst:114]:
```suggestion
    rule, and how partial derivatives of quantum circuits can be computed
```

File: videos.rst [hunk 2]
```diff
@@ -98,6 +98,42 @@ Sit back and explore quantum machine learning and quantum programming with our c
...
+
+    Explore the relationships between deep learning, differentiable
+    programming, and quantum computing. Learn how quantum circuits can be
+    trained just like neural networks!
+
+.. youtube:: McgBeSVIGus
+    :title: Automatic Differentiation of Quantum Circuits
+    :author: Nathan Killoran (Xanadu)
+
+    Dive into the world of quantum gradients. Learn how to apply the parameter shift
```
Comment (co9olguy) [inline videos.rst:113]:
```suggestion
    Dive into the world of quantum gradients. Learn how to apply the parameter-shift
```

File: videos.rst (hunk 1 above)
Comment (co9olguy) [inline videos.rst:114]:
```suggestion
    rules, and how derivatives of quantum circuits can be computed
```

**Comment by co9olguy** [comment]
Perhaps we should merge these now (without announcement---we've already shared these same videos in the past), and plan for a future update of the videos page? @glassnotes @josh146 what do you think?

**Comment by josh146** [comment]
Yep

**Pull request merged**